packages/data-context/src/sources/VersionsDataSource.ts

```diff
@@ -11,53 +19,102 @@ interface VersionData {
   latest: Version
 }
 
+const REMOTE_MANIFEST_URL = 'https://download.cypress.io/desktop.json'
+const NPM_CYPRESS_REGISTRY = 'https://registry.npmjs.org/cypress'
```

Comment:
I wonder how many users are on private npm registries? Guessing it's not super common.

Reply:
it is more common that you'd think. 
Every closed source stuff will be published locally. 
So in a lot of big corps with shared components, it really becomes attractive.
I used artifactory